docker/MOL/docker-compose.yml

```diff
@@ -0,0 +1,137 @@
+version: "2.4"
+
+volumes:
+  database:
+    name: mol-db
+  blockchain:
+    name: mol-data
+  tipbot-blockchain:
+    name: mol-tipbot-data
+  tipbot-database:
+    name: mol-tipbot-db
+
+services:
+
+  indexer:
+    container_name: mol-indexer
+    image: blockcore/indexer:0.2.39
+    mem_limit: 2048m
+    cpus: 1.500
+    environment:
+      VIRTUAL_HOST: mol.indexer.blockcore.net
+      VIRTUAL_PORT: 9910
+      VIRTUAL_PROTO: http
+      VIRTUAL_NETWORK: proxy
+      LETSENCRYPT_HOST: mol.indexer.blockcore.net
+      LETSENCRYPT_EMAIL: admin@blockcore.net
+      ASPNETCORE_URLS: https://+:9910
+      Logging__LogLevel__Default: Debug
+    command: ["--chain=MOL"]
+    restart: unless-stopped
+    depends_on:
+      mongo:
+        condition: service_healthy
+      chain:
+        condition: service_healthy
+    networks:
+      - molxnetwork
+      - proxy
+
+  chain:
+    container_name: mol-chain
+    image: blockcore/node-multi:1.1.50
+    mem_limit: 6144m
```

Comment:
This is a lot more than is really needed, but we'll tune it later. Probably would be fine with 3GB or 4GB.

Reply:
I made a copy of the current IMPLX settings. I will reduce